intel_extension_for_transformers/neural_chat/docs/notebooks/amp_optimization_on_spr.ipynb

```diff
@@ -62,7 +62,7 @@
    "source": [
     "from intel_extension_for_transformers.neural_chat import build_chatbot\n",
     "from intel_extension_for_transformers.neural_chat.config import PipelineConfig, AMPConfig\n",
-    "config = PipelineConfig(optimization_config=AMPConfig())\n",
```

Comment:
shall we use Intel/NeuralChatv1.1 for all the places using MPT?

Reply:
ok, I will update.